src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -131,34 +127,6 @@ exports[`StoryPromo Container should render correctly for amp 1`] = `
   }
 }
 
-@media (min-width:20rem) and (max-width:37.4375rem) {
```

Comment:
Snapshots have lost font stuff, likely as the tests arnt running inside the service config, can you pop them inside it please?

Reply:
Good spot!